.github/workflows/release.yml

```
@@ -0,0 +1,64 @@
+      matrix:
+        include:
+          - os: ubuntu-latest
+            name: "Linux (.NET 8/9)"
+          - os: windows-latest  
+            name: "Windows (All Frameworks)"
+    
+    runs-on: ${{ matrix.os }}
+    name: Build - ${{ matrix.name }}
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)